crash_similarity.py

```diff
@@ -15,9 +15,29 @@
 import numpy as np
 import bisect
 import pyximport; pyximport.install()
-
+import datetime
 # import download_data
 
+# Checks if the model has been trained in the last 24 hours (using datetime.timedelta class)
+def check_if_model_trained_in_last_24hours(time_interval):
```

Comment:
>  Its better to pass time interval as a parameter.

I meant that it will be good to pass `time_interval` as time from last update. E.g : in future we will want to check if the model was trained in the last six hours